validator/src/main.rs

```diff
@@ -1287,6 +1287,14 @@ pub fn main() {
                 .default_value(&default_rpc_threads)
                 .help("Number of threads to use for servicing RPC requests"),
         )
+        .arg(
+            Arg::with_name("rpc_timeout")
+                .long("rpc-timeout")
+                .value_name("NUMBER")
+                .validator(is_parsable::<u64>)
+                .takes_value(true)
```

Comment:
Default to "30" seconds?

Reply:
:+1: 

done https://github.com/solana-labs/solana/pull/14657/commits/2b73a6c1ce44ba1511c66fcd6be431d6999f46f4

Originally this flag is also used for the blockstore code-path, not only the bigtable codepath of getConfirmedBlocks. I wanted to avoid incompatible change of timing out at 30 secs by default.  But, later I noticed that bigtable is actually causing trouble. I reverted the changes for the blockstore's codepath.

So, assuming bigtable's user is only us, I think it's safe to change the behavior now. :)